crates/hir_ty/src/display.rs

```diff
@@ -1141,7 +1141,7 @@ impl HirDisplay for Path {
                 write!(f, ">")?;
             }
             (_, PathKind::Plain) => {}
-            (_, PathKind::Abs) => write!(f, "::")?,
+            (_, PathKind::Abs) => write!(f, "")?,
             (_, PathKind::Crate) => write!(f, "crate")?,
```

Comment:
Any idea why this wasn't working?

Reply:
If I still keep `::`, then the absolute path will be displayed in a way that starts with `::::`.

Reply:
The `write!(f, "")` is unnecessary though, you can just make it `{}`

Reply:
okay

Reply:
No, I meant the line below (`(_, PathKind::Crate) => write!(f, "crate")?`).

Reply:
Never mind, I was a bit confused.